### PR TITLE
Getting ready for pypi, fixes setup_cuda.py

### PR DESCRIPTION
**Pull request by bkmi**
I don't know if I like the multiple setup.py though. If we decide to go with it, I believe this is a solution. I'm attempting the install now.

### CODE REVIEW

**Comment by bkmi** [comment]
In fact, I get an error when attempting to use the `setup_cuda.py`. I don't think it is possible to use python_cuda.py it fails at gcc.

**Comment by bkmi** [comment]
setup_cuda.py had a bug from origin/master.

File: pack.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError("Unable to find version string.")
+
+
+def setup_kwargs(**kwargs):
+    return kwargs
+
+
+KWARGS = setup_kwargs(
```
Comment (mariogeiger) [inline pack.py:29]:
```suggestion
KWARGS = dict(
```

File: setup_cuda.py [hunk 2]
```diff
@@ -1,31 +1,18 @@
...
+from setuptools import setup
 from torch.utils.cpp_extension import BuildExtension, CUDAExtension
+from pack import KWARGS
 
 ext_modules = [
     CUDAExtension('e3nn.rsh',
-                  sources=['src/rsh/rsh_bind.cpp',
-                           'src/rsh/rsh_cuda.cu'],
+                  sources=['src/real_spherical_harmonics/rsh_bind.cpp',
+                           'src/real_spherical_harmonics/rsh_cuda.cu'],
```
Comment (mariogeiger) [inline setup_cuda.py:9]: Oops sorry

**Pull request merged**